_recipes/lists-by-number-of-items.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Lists by number of items
+layout: site
+body_class: recipe
+keywords: reading lists, items, list analytics, list quality
+description: get information on lists and modules by the number of items or sections they contain.
+---
+
+## Background
+This recipe will sort all lists at your university by the number of items they contain.  The sort of questions you might answer are:
+
+* How many lists have zero or very few items?
+* This may represent lists which need more engagement by list developers to provide useful resources for students.
+* Do any lists have particularly high numbers of items?
+* This may indicate lists which are over-enriched with content, and which may discourage student engagement via information overload.
```

Comment:
```suggestion
  * This may indicate lists which are over-enriched with content, and which may discourage student engagement via information overload.
```